EIPS/eip-eth-vhash.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: TBD
+title: eth/vhash - blob-aware mempool
+description: Make mempool messaging vhash aware
+author: Csaba Kiraly (@cskiraly)
+discussions-to: 
```

Comment:
Please create a discussions topic in Eth Magicians using the template
https://ethereum-magicians.org/c/eips/5